cmake/modules/BPF.cmake

```
@@ -11,7 +11,7 @@ set(LLC "llc")
 set(LLVM_STRIP "llvm-strip")
 set(BPFTOOL "bpftool")
 set(BTF_FILE "/sys/kernel/btf/vmlinux")
-option(USE_BUILTIN_VMLINUX "Whether or not to use the builtin vmlinux.h for building the BPF programs instead of trying to generate one from the system" True)
+option(USE_BUILTIN_VMLINUX "If true, use the builtin vmlinux.h for building BPF probes instead of generating one from system BTF" True)
```

nit:
```suggestion
option(USE_BUILTIN_VMLINUX "If true, use the builtin vmlinux.h for building eBPF probes instead of generating one from system BTF" True)
```

also, not in scope for this PR, but IIRC we decided to always use the builtin vmlinux.h (https://github.com/elastic/ebpf/issues/69), right?